sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClientBuilder.java

```diff
@@ -159,7 +159,7 @@ public KeyAsyncClient buildAsyncClient() {
      * @return the updated ServiceClientBuilder object.
      * @throws IllegalArgumentException if {@code endpoint} is null or it cannot be parsed into a valid URL.
```

Comment:
Include the exception object in the logger here.
Same as [this](https://github.com/Azure/azure-sdk-for-java/pull/5827#discussion_r334657507)

Reply:
done